index.html

```diff
@@ -43,8 +43,8 @@
         // editors, add as many as you like
         // only "name" is required
         editors: [
-          { name: "Andrew Hughes", url: "FIXME_PERSONAL_URL",
-            company: "", companyURL: "FIXME_COMPANY_URL" },
+          { name: "Andrew Hughes", url: "https://idimmusings.com/",
+            company: "ITIM Consulting", companyURL: "https://idimmusings.com/" },
```

Comment:
Hmm, don't know if you want to use your personal website as the company one or vice versa... are you sure?

Reply:
Yes, I'm sure.